actions/lint-insights/action.yaml

```diff
@@ -0,0 +1,36 @@
+name: 'Lint Security Insights File'
```

Comment:
Nit: https://docs.github.com/en/actions/sharing-automations/creating-actions/creating-a-composite-action#creating-a-composite-action-within-the-same-repository suggests that composite actions used within the repository should live in `.github/actions`

Reply:
You bring up a great point. From the original comment, I was thinking that the action may be imported in CI workflows in other repositories. However, maybe that makes more sense if it was located in the `ossf/security-insights-spec` or `ossf/si-tooling` repository.